plugins/mimecast/komand_mimecast/tasks/monitor_siem_logs/task.py

```diff
@@ -126,7 +168,7 @@ def _filter_and_sort_recent_events(
             self.logger.info("None of the raw logs returned from Mimecast where with in filter timerange")
         if task_output:
             self.logger.info(f"Latest event time returned from Mimecast logs: {latest_time}")
-        return task_output
+        return task_output, last_log_line
```

Comment:
minor we should update the typing on `_filter_and_sort_recent_events` now we return a tuple

Reply:
I have updated the type hinting an the methods doc string to include this new variable